openssl/src/aes.rs

```diff
@@ -0,0 +1,112 @@
+use ffi;
+use std::mem;
+use libc::c_int;
+
+use symm::Mode;
+
+#[derive(Debug)]
+pub struct KeyError(());
```

Comment:
These functions don't adjust the error stack: https://github.com/openssl/openssl/blob/master/crypto/aes/aes_core.c#L640-L643